packages/kbn-scout-reporting/src/cli/upload_events.ts

```diff
@@ -20,13 +21,13 @@ export const uploadEvents: Command<void> = {
     string: ['eventLogPath', 'esURL', 'esAPIKey'],
     boolean: ['verifyTLSCerts'],
     default: {
-      esURL: process.env.ES_URL,
-      esAPIKey: process.env.ES_API_KEY,
+      esURL: SCOUT_REPORTER_ES_URL,
+      esAPIKey: SCOUT_REPORTER_ES_API_KEY,
     },
     help: `
     --eventLogPath    (required)  Path to the event log to upload
-    --esURL           (required)  Elasticsearch URL [env: ES_URL]
-    --esAPIKey        (required)  Elasticsearch API Key [env: ES_API_KEY]
+    --esURL           (required)  Elasticsearch URL [env: SCOUT_REPORTER_ES_URL]
+    --esAPIKey        (required)  Elasticsearch API Key [env: SCOUT_REPORTER_ES_API_KEY]
     --verifyTLSCerts  (optional)  Verify TLS certificates
```

Comment:
Did you not mean to extend this CLI with an ENV parameter, like in `packages/kbn-scout-reporting/src/cli/initialize_report_datastream.ts`?

Reply:
Good catch. 👍🏽

Reply:
Resolved in https://github.com/elastic/kibana/pull/206431/commits/8bc870398d6ca0217bab5d9d2065e2085c3612a1